### DISCUSSION TITLE
Load model from g2pW/g2pw.onnx failed:Load model g2pW/g2pw.onnx failed.

### DISCUSSION THREAD

**Discussion by andymethane** [fishaudio/Bert-VITS2#317] [Q&A] [answered]
运行progress_text.py时显示没有g2pw.onnx文件，这个要在哪里生成呢？

**Comment by andymethane** [comment]
注：是在运行Extra-Fix版本中遇到的问题

**Comment by yihuitang** [comment, marked as answer]
在这里下载：https://openi.pcl.ac.cn/Stardust_minus/Bert-VITS2/modelmanage/model_filelist_tmpl?name=G2PWModel-v2-onnx

**Reply by andymethane** [reply]
找到了，太感谢了！！

**Reply by yqlmzydxn** [reply]
请问解决了吗，我的试了下还是不行
